SKILL.tsv

```diff
@@ -1089,35 +1089,35 @@ SKILL_20150317_001088	Fire Ball
 SKILL_20150317_001089	{#993399}{ol}[Magic] - [Fire]{/}{/}{nl} Summon Fire sphere. You can Blow the sphere using sword attack.
 SKILL_20150317_001090	Attack #{SkillAtkAdd}#{nl}Attack #{SkillSR}# times{nl}Duration #{CaptionTime}# seconds
 SKILL_20150317_001091	Fire Wall
-SKILL_20150317_001092	{#993399}{ol}[Magic] - [Fire]{/}{/}{nl} Create flame barrier to damage adjacent enemies.
+SKILL_20150317_001092	{#993399}{ol}[Magic] - [Fire]{/}{/}{nl} Creates a flame barrier inflicting damage to a nearby enemy.
 SKILL_20150317_001093	Attack #{SkillAtkAdd}#{nl}#{CaptionRatio}# Fire Wall #{nl}Attack #{CaptionRatio2}# times{nl}Duration 15 seconds
 SKILL_20150317_001094	Enchant Fire
-SKILL_20150317_001095	Grant Fire Property attack force to target. Additional damage is applied for basic attack.
+SKILL_20150317_001095	Grants the Fire attack attribute to the target. Additional damage applies to your basic attack.
```

Comment:
Unnecessary definite article (the fire attack).
Also, in light of elementals being likely changed to Property instead of Attribute - "Grants/Adds Fire Property to target's attacks"?